samigo/samigo-app/src/webapp/jsf/delivery/item/deliverCalculatedQuestion.jsp

```diff
@@ -95,3 +95,47 @@ should be included in file importing DeliveryMessages
       escape="false" />
   </h:panelGroup>
 </h:panelGroup>
+
+<script>
```

Comment:
It's not an ideal solution to add js in this way I would prefer this be in a js file that is included vs inlining.

Reply:
Agreed, I'll take a look in the next couple of days

Reply:
I moved the js to separate files for numeric and calculated question, just had to leave the error message declaration in the jsp for it to work correctly.